### PR TITLE
{lang}[GCCcore/6.4.0] CLISP v2.49

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 3 out of 5 (1 easyconfigs in this PR)
ca004.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2c9ed7cd2e79ab546b81b32f2563b0ae for a full test report.

File: easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+description = """
+ Common Lisp is a high-level, general-purpose, object-oriented, dynamic,
+ functional programming language.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_BZ2]
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-6.4.0.eb]: In my testing, a `404` error when trying to download these sources

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
ca004.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e0269c265921eb71fe936ff44fceb489 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
ca004.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ab6ea3ad92006571402198bf3209094a for a full test report.

File: easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -32,7 +32,7 @@ dependencies = [
 ]
 
 prebuildopts = """
-    sed -e 's/ac_cv_have_decl_rl_readline_state" = x""yes/ac_cv_have_decl_rl_readline_state" = x""JKP/'\
+    sed -e 's/ac_cv_have_decl_rl_readline_state" = x""yes/ac_cv_have_decl_rl_readline_state" = x""FOOBAR/'\
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-6.4.0.eb]: What's the reason for this one?

**Comment by verdurin** [comment]
Looks good now - thanks for the patch.

**Comment by verdurin** [comment]
Merging, thanks @JackPerdue

File: easybuild/easyconfigs/c/CLISP/patch-modules_readline_readline.lisp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+$NetBSD: patch-modules_readline_readline.lisp,v 1.1 2016/09/20 14:10:25 wiz Exp $
```
Comment (boegel) [inline easybuild/easyconfigs/c/CLISP/patch-modules_readline_readline.lisp]: @JackPerdue where did you get this patch from exactly, do you have an URL for it?

Comment (boegel) [inline easybuild/easyconfigs/c/CLISP/patch-modules_readline_readline.lisp]: ah, it's mentioned in the easyconfig file, nevermind

**Pull request merged**